sros2_cmake/package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+    <package format="2">
+    <name>sros2_cmake</name>
+    <version>1.0.0</version>
+    <description>Cmake macros to configure security for nodes</description>
+    <author email="aws-b9-platform@amazon.com">AWS B9 Team</author>
+    <maintainer email="aws-b9-platform@amazon.com">AWS B9 Team</maintainer>
+    <license>Apache 2.0</license>
+    
+    <buildtool_depend>cmake</buildtool_depend>
+    <export>
```

Comment:
I believe we need `<build_depend>` and `<build_export_depend>` tags for `sros2` since we need that package to be built in order to run the cmake macros from other packages.